gallery.html

```diff
@@ -0,0 +1,186 @@
+<!doctype html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <title> Project Gallery </title>
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Lato&family=Mali:wght@600&family=Zilla+Slab:wght@700&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+
+    <main class="gallery-container" id="gallery-container">    
+      <h1 class="gallery-title"> Project Gallery </h1>
+        <div class="articles-container">
+         <div class="article-wrapper">
+          <img src="main-img.jpg" class="gallery-img"/>
+           <div class="article-content">
+            <h2 class="article-heading"> Sarah Thompson </h2>
+              <h3 class="project-name"> Project: "CodeConnect" </h3>
+               <p class="project-info"> 
+                 <span class="desc-title"> Description: </span> 
+                  CodeConnect is a web application that helps 
+                  coding bootcamp students and graduates connect with mentors in their 
+                  field. It allows students and graduates to browse profiles of experienced 
+                  professionals, schedule mentorship sessions, and receive 
+                  guidance on career paths, projects, and technical concepts.
+               </p>
+             <a href="#"><button class="gallery-btn"> Read More </button></a>
```

Comment:
I'm curious as to why you are wrapping a button inside of an anchor tag? I know this is just a draft, but where would 'Read More' direct or how would it change the display?